web/templates/twig-source/incuna-surveys/fields/free-text.html

```diff
@@ -1,3 +1,3 @@
-<div drf-form-field="to.fieldOptions" class="text" field-id="options.key">
+<div drf-form-field="to.fieldOptions" class="text {% raw %}{{ model[to.fieldSetId][options.key] ? 'not-empty' : '' }}{% endraw %}" field-id="{% raw %}{{ to.autoId }}{% endraw %}">
```

Comment:
oh you can't `ng-class` because the directive is already doing it. I dislike directives which use `replace` for this reason!